doc/source/data/creating-datasets.rst

```diff
@@ -328,6 +328,19 @@ from a local filesystem:
   to ``ray/data/examples/data`` directory in the Ray package. This scheme is used
   only for testing or demoing examples.
 
+Reading Compressed Files
+==========================
+
+Ray Datasets supports reading compressed files using the ``arrow_open_stream_args`` arg.
+`Codecs supported by Arrow <https://arrow.apache.org/docs/python/generated/pyarrow.compress>`__
+(brotli, gzip, lz4, lz4_raw, snappy, zstd) are compatible with Ray Datasets.
```

Comment:
can you use the same link as [`read_tfrecords`](https://anyscale-ray--33313.com.readthedocs.build/en/33313/data/api/doc/ray.data.read_tfrecords.html) - https://arrow.apache.org/docs/python/generated/pyarrow.CompressedInputStream.html . Also stick with same for what PyArrow.CompressedInputStream supports - “bz2”, “brotli”, “gzip”, “lz4” or “zstd”.